css/values.css

```diff
@@ -0,0 +1,336 @@
+	width: 50%;
+}
+
+/* general card styling */
+.values-card {
+	padding: 60px;
+	color: var(--text-color-light);
+	border-radius: var(--border-radius);
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+/* text layout*/
+.values-card .text {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* card specific */
+.humanity {
+	background-color: var(--color-copper);
+}
+
+.sustainability {
+	background: linear-gradient(
+			269deg,
+			#0e0e0e 46.26%,
+			rgba(27, 27, 27, 0) 99.04%
+		),
+		url("/assets/images/pages/values/sustainability.png") top/cover;
+}
+
+.authenticity {
+	background: linear-gradient(
+			88deg,
+			#b91515 47.26%,
+			rgba(185, 21, 21, 0.73) 55.57%,
+			rgba(119, 45, 45, 0.4) 68.47%
+		),
+		url("/assets/images/pages/values/authenticity.png") top/cover;
+}
+.multigenerational {
+	background-color: var(--color-grape);
+}
+.education {
+	background: linear-gradient(
+			90deg,
+			#184f85 63.23%,
+			rgba(24, 79, 133, 0) 83.39%
+		),
+		url("/assets/images/pages/values/education.png") top/cover;
+}
+
+.innovation {
+	background: linear-gradient(
+			280deg,
+			#007a80 67.16%,
+			rgba(0, 122, 128, 0) 80.36%
+		),
+		url("/assets/images/pages/values/innovation.png") top/cover;
+}
+
+/* 
+===================================
+Support Banner
+===================================
+*/
+.support {
+	background-color: var(--color-blue);
+}
+
+/* text adjustments */
+.support h2 {
+	color: var(--color-light-amaranth);
+}
+
+.support p {
+	color: var(--text-color-light);
+	font-size: 46px;
+	font-weight: 700;
+	line-height: 50px;
+}
+
+.support a {
+	width: fit-content;
+}
+
+.support-text {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* layout */
+.support .wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-med);
+}
+
+/* heading and heart layout*/
+.support-heading {
+	display: flex;
+	gap: var(--spacing-sm);
+}
+
+.support-heading img {
+	width: var(--spacing-med);
+}
+
+/* support un images */
+.support-img-container {
+	display: flex;
+	flex-wrap: wrap;
+	gap: var(--spacing-sm);
+}
+
+.support-img-container img {
+	width: fit-content;
+	max-height: 100px;
+}
+/* 
+===================================
+Empowering 
+===================================
+*/
+/* using styled img container component in global.css */
+
+.empowering .wrapper {
+	flex-direction: row-reverse;
+}
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+===================================
+*/
+
+/* Tablet and smaller 1024px*/
+@media (max-width: 1024px) {
+	/* 
+===================================
+Values
+===================================
+*/
+	.values-img-text .img-mobile {
+		display: block;
+		visibility: visible;
+	}
+	/* layout to colum */
+	.values-row,
+	.values-img-text {
+		flex-direction: column;
+	}
+
+	.values-card {
+		width: 100%;
+		padding: var(--spacing-sm);
+		min-height: 300px;
```

Comment:
On Mobile: titles should be centered and spacing looks off. Maybe replace min-height with padding.

<img width="581" alt="Screen Shot 2024-01-11 at 3 04 00 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/a3a98512-46f4-429e-82c3-da4de3d76413">

Reply:
For titles centered I have an MQ at 470px w text align centering at line 335.

At the same MQ I have adjusting the min height to better match the spacing of the value cards to the figma design - please let me know if that works!

![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/8f895b85-4a2c-409c-aa6b-aa34f7b52e7c)

Reply:
That works!